apps/ledger-live-mobile/src/newArch/features/ModularDrawer/components/GenericError.tsx

```diff
@@ -0,0 +1,51 @@
+import { Flex, Icons, Text, Button } from "@ledgerhq/native-ui";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components/native";
+
+type Props = { onClick?: () => void; type: "backend" | "internet" };
+
+export const GenericError = ({ onClick, type }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <Container p={6} flexDirection="row">
+      <Icons.DeleteCircleFill size="M" color={"error.c70"} />
+      <Content ml={3}>
+        <Title mb={2}>{t("modularDrawer.errors.title")}</Title>
+        <Description mb={4}>{t(`modularDrawer.errors.${type}`)}</Description>
+
+        {onClick && (
+          <Button type="error" onPress={onClick} alignSelf="flex-start">
```

Comment:
Probably out of scope for this ticket, but it feels like the Button component should have styling for onPress - otherwise there's no feedback that the retry button does anything (if subsequent requests are also unsuccessful)

Reply:
In the case of no internet I wasn't able to see a loading page, so there wasn't visible feedback that the button was doing anything. but maybe that's fine - the error is clear about what the issue is.